src/app/detachable-component/detachable-component.component.html

```diff
@@ -0,0 +1,12 @@
+<h2>Detachable example</h2>
+
+<p><strong>Note:</strong> Open the console for additional logs.</p>
+
+	<h3>Detachable editor:</h3>
+	<div #container>
+		<div #editor>
+			<ckeditor data="<p>Hi, I am CKEditor 4!</p>" [config]="{ extraPlugins: 'divarea' }"></ckeditor>
```

Comment:
My final concern 🤔 

Why we can't use the default config? I mean - I tried it and it hang the e2e test 🤔  Could you tell if some awaits after reattach could possibly do the job? As I can see on the browser - the content is loaded - however, the test couldn't be finished.

Reply:
It hangs e2e test because default config uses editable inside the `iframe`. And it means that we'd need to pierce through the `iframe` and check content there. Honestly, I don't have even the slightest idea how to do it in Protractor and its docs aren't great either. So I did what was done in all our examples – used `divarea` :see\_no\_evil: